README.md

```diff
@@ -28,6 +28,14 @@ In your `package.json`, add these scripts:
 "crowdin:delete-branch": "mollie-crowdin delete-branch"
 ```
 
+## Branch name
+
+By default the `@mollie/crowdin-in` uses the current git branch name for the syncing to the Crowdin branch. It is possible to override that by defining a `CROWDIN_BRANCH_NAME` environment variable in the `.env` file
+
+```shell
+CROWDIN_BRANCH_NAME={your-branch-name}
+```
+
```

Comment:
Shouldn’t this rather be a flag for the respective command, rather than an env var?

Reply:
Same here, would expect this to be `--branch=master`. The tool has Crowdin in the name, so I think we can be concise 😊

Reply:
Thanks. What you both say makes perfect sense. I've changed it to be instead a `--branch-name` (or `-b`) option that you can pass to the `upload`, `download` and `delete-branch` commands.
```shell
mollie-crowdin download --branch-name my-custom-branch-name
```